samples/bluetooth/fast_pair/locator_tag/sample.yaml

```diff
@@ -26,3 +26,28 @@ tests:
       - thingy53/nrf5340/cpuapp
       - thingy53/nrf5340/cpuapp/ns
     tags: bluetooth ci_build sysbuild
+  sample.bluetooth.fast_pair.locator_tag.release:
+    sysbuild: true
+    build_only: true
+    integration_platforms:
+      - nrf52dk/nrf52832
+      - nrf52833dk/nrf52833
+      - nrf52840dk/nrf52840
+      - nrf5340dk/nrf5340/cpuapp
+      - nrf5340dk/nrf5340/cpuapp/ns
+      - nrf54l15pdk/nrf54l15/cpuapp
+      - nrf54l15pdk@0.3.0/nrf54l15/cpuapp
```

Comment:
Keep in mind to remove it after the ongoing Zephyr upmerge. Then we will test only the default revision of `nrf54l15pdk/nrf54l15/cpuapp` (0.3.0)